app/src/test/java/com/stevesoltys/seedvault/restore/install/ApkRestoreTest.kt

```diff
@@ -403,6 +416,24 @@ internal class ApkRestoreTest : TransportTest() {
         }
     }
 
+    @Test
+    fun `no apks get installed when blocked by policy`(@TempDir tmpDir: Path) = runBlocking {
+        every { installRestriction.isAllowedToInstallApks() } returns false
+        every { storagePlugin.providerPackageName } returns storageProviderPackageName
+
+        apkRestore.restore(backup).collectIndexed { i, value ->
+            when (i) {
+                0 -> {
+                    // single package fails without attempting to install it
```

Comment:
@grote I didn't get this part, why the single package?

Reply:
because we are just testing with a single package here

Reply:
What I don't get is where that one package is coming from / what package would that be.

I do see that we expect it to fail installing, that makes sense.

Reply:
Follow the traces. We call `apkRestore.restore(backup)`.  The parameter is `val backup = RestorableBackup(metadata.copy(packageMetadataMap = packageMetadataMap))`. Then `packageMetadataMap = hashMapOf(packageName to packageMetadata)`, so only a single package being used for this test.